importer/mappings/de-he_(de).json

```diff
@@ -0,0 +1,21 @@
+{
+    "country": {
+        "item": "Q40",
```

Comment:
Q183.

Since the preview is correct I'm wondering if you maybe have local uncommitted changes?

Reply:
True, missed adding it to a committ...